index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Lab 1</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="MichelleP64" content="MichelleP64">
+    
+    <p>MichelleP64/p>
+</head>
+<body>
```

Comment:
It looks like this may have been shifted into the wrong spot
```suggestion
</head>
<body>
    <p>MichelleP64/p>
```